README.md

```diff
@@ -2,12 +2,17 @@
 
 Official nodejs bindings for [Razorpay API](https://docs.razorpay.com/docs/payments).
 
+Read up here for getting started with Razorpay: https://docs.razorpay.com/docs/getting-started  
+Read up here for understanding the payment flow in Razorpay: https://docs.razorpay.com/docs/payment-flow
+
 ## Installation
 
 ```bash
 npm i razorpay --save
 ```
 
+NPM Link: https://www.npmjs.com/package/razorpay
```

Comment:
@akshaybhalotia let's use the compact npm badge <https://nodei.co/#razorpay> instead